src/js/components/FormField/FormField.js

```diff
@@ -184,12 +184,30 @@ const FormField = forwardRef(
       required,
       style,
       validate,
+      validation,
```

Comment:
Hey @joaoGabriel55 ! Thanks for working on this ticket. Instead of introducing a new prop `validation`, we'd instead like to extend the current functionality of `validate` to also allow for `max` and `threshold` (I noticed I have a mistype in my original issue where I wrote `validation`, but I meant `validate`).

Can we update this PR to remove `validation` prop and just handle the functionality under `validate` ?

Reply:
yeah! Thanks for your explanation